src/phoenix/trace/tracer.py

```diff
@@ -17,26 +28,44 @@ def export(self, _: Any) -> None:
 
 
 _DEFUNCT_MSG = (
-    "DEFUNCT: `Tracer` is a defunct class in the current version of Phoenix. "
-    "It no longer has any purpose or functionality and will be removed in the future."
+    "DEFUNCT: `Tracer` is now a defunct class in the current version of Phoenix and is being "
+    "deprecated. It no longer has any purpose or functionality and will be removed in the future."
 )
+
 _USE_ENV_MSG = (
-    "Setting endpoint through the HttpExporter is no longer supported. "
-    'Use environment variables instead, e.g. os.environ["PHOENIX_PORT"] = "54321"'
+    "Setting the Phoenix endpoint through the HttpExporter is no longer supported. "
+    'Please use environment variables instead, e.g. os.environ["PHOENIX_PORT"] = "54321"'
+)
+
+_ON_APPEND_DEPRECATION_MSG = (
+    "OpenInference has been updated for full OpenTelemetry compliance. The ability to set "
+    "`on_append` callbacks are removed."
 )
 
 
 class Tracer:
     _exporter: Any
 
-    def __init__(self, *args: Any, **kwargs: Any) -> None:
-        for arg in args:
-            if isinstance(arg, HttpExporter):
-                logger.warning(_USE_ENV_MSG)
-        if "exporter" in kwargs:
+    def __init__(
+        self,
+        exporter: Any = None,
+        on_append: Optional[Callable[[List[Span]], None]] = None,
+    ) -> None:
+        if exporter is not None:
             logger.warning(_USE_ENV_MSG)
+        self.exporter = _OpenInferenceExporter().otel_exporter
         logger.warning(_DEFUNCT_MSG)
 
+        if on_append is not None:
+            logger.warning(_ON_APPEND_DEPRECATION_MSG)
+
+        self._tracer_provider = trace_sdk.TracerProvider(resource=Resource(attributes={}))
+
+    def _configure_otel_tracer(self) -> None:
+        span_processor = SimpleSpanProcessor(span_exporter=self.exporter)
+        self._tracer_provider.add_span_processor(span_processor)
+        trace_api.set_tracer_provider(tracer_provider=self._tracer_provider)
+
```

Comment:
This class is defunct. Let's not add any more code to it.
```suggestion
        logger.warning(_DEFUNCT_MSG)
```

Reply:
addressed in 7d0f3c39